src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -193,6 +193,11 @@ In this message, the phrase "should not be reached" means that this error messag
     <value>Failed to get the Java SDK version. Please ensure you have Java {0} or above installed.</value>
     <comment>{0} - The Java version number</comment>
   </data>
+  <data name="XA0035" xml:space="preserve">
+    <value>Unable to determine the Android ABI from the value '{0}'. Set $(RuntimeIdentifiers) to valid identifiers for the Android platform.</value>
```

Comment:
It's not too important in this case since I bet the wording will be updated again as the .NET 6 story evolves, but for an extra margin of safety, I'll suggest making the initial wording:

```
<value>Unable to determine the Android ABI from the value '{0}'. Edit the project file in a text editor and set the 'RuntimeIdentifiers' MSBuild property to contain only valid identifiers for the Android platform.</value>
```

Motivation: Give users across a range of familiarity with MSBuild and across all the different translations of the message the best chance at finding the `RuntimeIdentifiers` MSBuild property (a lesson I learned recently in <https://github.com/xamarin/xamarin-android/commit/91022a97214bc6611e61354f723e3b47a0b119e0>).